build-config/scripts/onie-mk-itb.sh

```diff
@@ -139,6 +141,7 @@ DTB="$(realpath $DTB)"
 			arch = "$ARCH";
 			os = "linux";
 			compression = "none";
+			load = <$FDT_LOAD>;
```

Comment:
Right.  Maybe not `$10`, but perhaps `$8`, pushing the other arguments down by one.

Do not forget about `powerpc`, which also uses this script.  We need to have a good default for `powerpc`, `armv7` and `armv8`.

I would look at the source code for `dtc` and see what it does when `load` is not specified for a `dtb` stanza.